### PR TITLE
CVE-2016-3087

### PR DESCRIPTION
**Pull request by Bgood12**
Pull request for CVE-2016-3087.yml

### CODE REVIEW

File: cves/CVE-2016-3087.yml [hunk 1]
```diff
@@ -207,4 +226,6 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those in the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    A mistake was made during implementation. The developer simply did not validate
+    user input.
```
Comment (mesternefeld) [inline cves/CVE-2016-3087.yml]: I would suggest to add more to this section to make it more interesting.

File: cves/CVE-2016-3087.yml [hunk 2]
```diff
@@ -144,12 +156,19 @@ interesting_commits:
     emerging themes?
     If there are no interesting commits, demonstrate that you completed this section
     by explaining what happened between the VCCs and the fix.
-  answer:
+  answer: |
+    Git log only returned two commits betwwen the vcc and the fix. The latter
```
Comment (Chrestopher) [inline cves/CVE-2016-3087.yml]: Misspelling for the word between. I also think that using the terms latter and former can be confusing to some people who aren't too familiar with that kind of speech.

File: cves/CVE-2016-3087.yml [hunk 3]
```diff
@@ -164,38 +185,38 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies:
-    note:
+    applies: false
```
Comment (rburchlau001) [inline cves/CVE-2016-3087.yml:188]: I'm pretty sure you can just leave the ones you're not commenting blank

File: cves/CVE-2016-3087.yml [hunk 4]
```diff
@@ -207,4 +228,6 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those in the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    A mistake was made during implementation. The developer simply did not validate
```
Comment (rburchlau001) [inline cves/CVE-2016-3087.yml:232]: Where could this have been headed off? This could be punched up a bit to be more interesting.

**Pull request merged**